exporter/azure_monitor/azure_monitor.go

```diff
@@ -0,0 +1,46 @@
+package azure_monitor
+
+import (
+	"fmt"
+	"errors"
+	"go.opencensus.io/exporter/azure_monitor/common"
+	"go.opencensus.io/trace"
+)
+
+type AzureTraceExporter struct {
+	projectID          string
+	InstrumentationKey string

```

Comment:
Not sure about naming convention, do we want CamelCase here or not?

Reply:
For golang, attributes need to start with an uppercase to be made public for other files to use. I am currently going for CammelCase if that is fine with you. I will be more consistent next time.